tests/image-stream/tester.js

```diff
@@ -35,9 +34,14 @@ const doFilesContainTheSameData = (file1, file2) => {
 }
 
 const deleteIfExists = (file) => {
```

Comment:
Why should we test if the file exists?
Couldn't we just delete it in the try block?

Reply:
I think the reasoning behind it was that we want it to fail should the file not have been written, or not been written to that location.

Reply:
```javascript
return fs.unlinkAsync(file)
.catchReturn(function(error) {
  return (error.code === 'ENOENT')
})
```

Reply:
Otherwise somethig could happen to the file between `access` and `unlink`.

Reply:
Not sure I follow; both the access & unlink are in the `try` block

Reply:
You're right, this will work as expected.
It is just that the call to access is useless.
From the node fs.access docs:
>Using fs.access() to check for the accessibility of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file is not accessible.

Reply:
True that, fixing

Reply:
```javascript
return fs.unlinkAsync('klklklklklklklklklk')
.catch({ code: 'ENOENT' }, _.noop)
```

Reply:
Fair enough; updated!